cfpb/cfgov-refresh
# YES: content updates and print styles

Adds content updates to YES tools and enables printing in preparation for launch, currently scheduled for Friday, June 19.


## Additions

- Adds print CSS to YES car buying guide block. This CSS needs to be scoped to pages containing the block & can't be included in `main.css` because it hides general page content (`header`, `footer`, `.content_sidebar`, etc) on print.


## Changes

- Content updates for YES transportation tool and car buying guide block


## How to test this PR
Most of the content changes have been verified by the stakeholder on a demo server, so the main thing to check is the print functionality:

1. Check out branch
2. Run `yarn gulp`
3. Verify that the car buying guide print CSS is loaded:
    - If you have a recent-ish database dump, you can navigate to [the car buying guide page](http://localhost:8000/admin/pages/13104/edit/preview/) locally
    - Check one of the options, click the print button, and verify that the print styles hide most of the page's content (screenshot 1 below)
4. Verify print functionality works on transportation tool:
    - Turn on `YOUTH_EMPLOYMENT_SUCCESS` flag
    - Navigate to [transportation tool](http://localhost:8000/practitioner-resources/resources-youth-employment-programs/transportation-tool/)
    - Fill out budget fields and two transportation options. Click "Review your plan", then "Print and save", and verify print preview (Screenshot 2, below)


## Screenshots

### Printing car buying guide
<img width="576" alt="Screen Shot 2020-06-16 at 9 40 54 AM" src="https://user-images.githubusercontent.com/778171/84803638-1a19dd00-afb7-11ea-9e90-7ee29972c2ec.png">



### Printing transportation tool
<img width="579" alt="Screen Shot 2020-06-16 at 9 49 19 AM" src="https://user-images.githubusercontent.com/778171/84803648-1dad6400-afb7-11ea-85cd-d88417bb2dcb.png">


## Notes and todos

- I added a new directory & task for this on-demand CSS since the main ondemand task creates non-responsive versions of files. 


## Checklist

  - PR titles are used to generate the change log in [releases](../../releases),
    so good ones make that easier to scan.
  - Consider prefixing with the feature/section you're addressing,
    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"

#### Browser testing

Visually tested in the following supported browsers:
